test/cluster/dtest/alternator/__init__.py

```diff
@@ -0,0 +1,5 @@
+#
+# Copyright (C) 2025-present ScyllaDB
+#
+# SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
```

Comment:
This copyright line is fine for this file (the year is anachronistic, but it's fine), but please remember when porting other dtests that many of them were actually derived from Cassandra's tests so you can't just tack our own copyright statement and license on those files.